components/preparing_workspace/preparing_workspace.tsx

```diff
@@ -242,94 +198,6 @@ export default function PreparingWorkspace(props: Props) {
 
     const sendForm = async () => {
         const sendFormStart = Date.now();
-        setSubmissionState(SubmissionStates.Submitting);
-        try {
-            dispatch(savePreferences(user.id, [
```

Comment:
I imagine that we want the onboarding task list to always show boards and playbooks if they are installed now. To do that, we'll need to edit [onboarding_tasks_manager.tsx](https://github.com/mattermost/mattermost-webapp/blob/master/components/onboarding_tasks/onboarding_tasks_manager.tsx#L118-L123) to not check this preference, and only check if they are installed.

Reply:
yes, we should remove that validation.